bin/eslint-dependency-graph.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const {CLIEngine} = require('eslint')
+
+// TODO: Figure out how to deactive other rules.
+let cli = new CLIEngine({
+  rules: {
+    'github/dependency-graph': 1
+  }
+})
+cli.executeOnFiles(process.argv.slice(2))
+
+// TODO: Figure out how to deactive other rules.
+cli = new CLIEngine({
+  rules: {
+    'github/unused-export': 2,
+    'github/unused-module': 2
```

Comment:
Is it possible to use `warn/error` keywords instead of magic numbers here?

Reply:
Good call, I believe those work in this context. I'll make that change.